concepticondata/concepticon.tsv

```diff
@@ -31,7 +31,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 30	DISPEL	Motion	Force to go away.	Action/Process	
 31	PAN	Food and drink	Cooking utensil relatively flat, having a handle, used to cook food such as meat, vegetables or eggs, usually in oil or butter.	Person/Thing	
 32	EARWAX	The body	A yellowish, waxlike secretion from certain glands in the external auditory canal.	Person/Thing	
-33	NETTLE	Agriculture and vegetation	The common name for between 30-45 species of flowering plants with a cosmopolitan though mainly temperate distribution. They are mostly herbaceous perennial plants, but some are annual and a few are shrubby.	Person/Thing	
+33	NETTLE	Agriculture and vegetation	Any of numerous plants having stinging hairs that cause skin irritation on contact (especially of the genus Urtica or family Urticaceae) (Source: BabelNet)	Person/Thing	
```

Comment:
We often do not really use that "source". But we link to the metadata instead. Have a look at concept_set_meta, where you find a folder with wikipedia and babelnet, where you could add the id for this. The former translatoin comes probably from wikipedia.

Reply:
I see - many definitions are sourced like I did, that's why I assumed it to be the standard method. I assume I only need to add a source if my definition comes from a source not in that folder?

Reply:
They have shortcuts, like [WK:float](en.wikipedia.de/wiki/float), but your source doesn't have the ID. If you add [B:ID] this is also fine, maybe even better, and later, we can link them.